mqlc/mqlc_test.go

```diff
@@ -1271,10 +1271,6 @@ func TestCompiler_ResourceExpansion(t *testing.T) {
 				Type:    string(types.Range),
 				Binding: (3 << 32) | 1,
 			}, res.CodeV2.Blocks[2].Chunks[3])
-			assertFunction(t, "content", &llx.Function{
```

Comment:
This field has been removed so we can't use it for mqlc testing anymore

Reply:
nice, no harm removing it, thank you!